plugins/flytekit-kf-pytorch/flytekitplugins/kfpytorch/task.py

```diff
@@ -137,6 +138,7 @@ class Elastic(object):
     monitor_interval: int = 5
     max_restarts: int = 0
     rdzv_configs: Dict[str, Any] = field(default_factory=dict)
+    run_policy: Optional[RunPolicy] = field(default_factory=lambda: None)
 
```

Comment:
For the `rdzv_configs` which defaults to an empty dict, using a default factory is required to ensure that each instance gets a unique copy of the mutable default value.

I don't think anything speaks against using `run_policy: Optional[RunPolicy] = None` though, or am I overlooking something here?

Reply:
Addressed